code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -18,7 +18,8 @@ Mineral Sheets
 GLOBAL_LIST_INIT(sandstone_recipes, list (
 	new /datum/stack_recipe("pile of dirt", /obj/machinery/hydroponics/soil, 3, time = 10, one_per_turf = TRUE, on_floor = TRUE),
 	new /datum/stack_recipe("sandstone door", /obj/structure/mineral_door/sandstone, 10, one_per_turf = TRUE, on_floor = TRUE),
-  	new /datum/stack_recipe("sandstone airlock assembly", /obj/structure/door_assembly/door_assembly_sandstone, 4, time = 50, one_per_turf = TRUE, on_floor = TRUE),
+  	new /datum/stack_recipe("sandstone airlock assembly", /obj/structure/door_assembly/door_assembly_sandstone, 4, time = 5 SECONDS, one_per_turf = TRUE, on_floor = TRUE),
+	new /datum/stack_recipe("sarcophagus", /obj/structure/closet/coffin/sarcophagus, 5, time = 1.5 SECONDS, one_per_turf = TRUE, on_floor = TRUE),
```

Comment:
```suggestion
	new /datum/stack_recipe("sarcophagus", /obj/structure/closet/coffin/sarcophagus, 5, time = 10 SECONDS, one_per_turf = TRUE, on_floor = TRUE),
```
I'd prefer this take excruciatingly long to build as well, it's not something that should be easily made due it 1) being a sarcophagus

Reply:
5 Seconds would be enough? 10 is too much if you compare to other recipes build time, that could use some adjustments (why is sandstone door instant to build?!).